storage/object/reference-content/ssl-tls-certificate.mdx

```diff
@@ -0,0 +1,174 @@
+
+The private key is also used by the host for generating digital signatures, while the public key is used by clients for verifying those signatures.
+
+### When and why do I need an SSL/TLS certificate for Edge Services?
+
+When you enable Edge Services, initially your bucket content is served through the standard Edge Services endpoint, e.g. `https://bucket-name.edge.scw.cloud`. Scaleway's own SSL/TLS certificate, which covers this subdomain, is used to establish the encrypted connection between client and host. If you do not want to customize the standard Edge Services endpoint, you do not need to worry about creating SSL/TLS certificates.
+
+However, if you choose to [customize your Edge Services endpoint with your own subdomain](/storage/object/how-to/get-started-edge-services/#how-to-configure-a-custom-domain), Scaleway's own SSL/TLS certificate cannot longer be used to establish encrypted connections to your subdomain. Client connections are now initially going to a different domain which needs to be "guaranteed" by its own certificate (despite the CNAME record for the subdomain pointing to the Scaleway endpoint). 
+
+Therefore, when you customize your Edge Services endpoint with a subdomain, you are prompted to upload an SSL/TLS certificate for that subdomain. This certificate is stored in [Scaleway Secret Manager](https://www.scaleway.com/en/docs/identity-and-access-management/secret-manager/quickstart/).
+
+## How to get a certificate
+
+In general, SSL/TLS certificates can either be **self-signed** (signed by the subject of the certificate, e.g. the owner of the domain) or **CA-signed** (signed by a third party **C**ertificate **A**uthority which is publicly trusted). 
+
+**Self-signed certificates cannot be used with Edge Services, all certificates must be signed by a CA.**
+
+To get an SSL/TLS certificate for your domain or subdomain, you need to generate a Certificate Signing Request (CSR) and submit it to a Certificate Authority (CA) for them to validate your domain, who then send you a signed certificate. You may be able to carry out this procedure via your hosting provider, or from the command line.
+
+Find below some useful guides from different CAs on how to obtain a certificate from them:
+
+- [Let's Encrypt: Getting Started](https://letsencrypt.org/getting-started/). Let's Encrypt is a popular, non-profit CA that provides certificates at no cost. Note that the certificates are limited to 90 day validity, and must be renewed (again, at no cost) after this period.
+- [ZeroSSL: Creating an SSL Certificates](https://help.zerossl.com/hc/en-us/articles/360060119373-Creating-an-SSL-Certificate). ZeroSSL is another CA, which provides 90-day certificates at no cost, or certificates of longer duration at a price.
+- [Sectigo: Compare Sectigo SSL Certificates](https://www.sectigo.com/ssl-certificates-tls/compare). Sectigo is a CA who provide paid-for certificates with a validity period of between 1 and 6 years.
```

Comment:
Do we want to link to a paid provider here? There are several, why Sectigo and not Comodo or any other?